x-pack/plugins/cases/public/components/all_cases/types.ts

```diff
@@ -24,3 +24,8 @@ export interface EuiBasicTableOnChange {
   };
   sort?: EuiBasicTableSortTypes;
 }
+export interface Solution {
```

Comment:
Would it be better for this type to live one level up in x-pack/plugins/cases/public/components/all_cases/types.ts? What do you think @cnasikas ?

Reply:
Yeah sorry, that's what I meant. I think it's used in separate components right? Or maybe I misunderstood the file structure?

Reply:
No, it is used only with `all_cases` folder